docs/admin/templates/managing-templates/change-management.md

```diff
@@ -88,6 +88,14 @@ coder templates push --yes $CODER_TEMPLATE_NAME \
     --name=$CODER_TEMPLATE_VERSION # Version name is optional
 ```
 
+You can also use the [coder/setup-action](https://github.com/coder/setup-coder)
+GitHub Action to install the Coder CLI and push new template versions.
```

Comment:
we use `You can also` at the beginning and end of this section. If this is an alternative to the steps above, can we move this sentence to the top of the section so that users can decide which path to take before we show them the steps?